README.md

```diff
@@ -106,17 +113,18 @@ To distinguish such patterns from traditional regular expression patterns, the
 encoding must start with a `{` or contain line breaks. When using just JSON
 encoding, backslashes must get quoted inside strings. When using YAML, this
 isn't necessary. The following pattern strings are equivalent:
+```go
```

Comment:
```suggestion
```
```

This isn't really go code here so I don't think this quite works.

Reply:
you're right, thanks for pointing it out!
adjusted